### DISCUSSION TITLE
Does openVSCode have a logo?

### DISCUSSION THREAD

**Discussion by Verhoeckx** [gitpod-io/openvscode-server#285] [Q&A]
Assuming that I eventually will be able to run openVSCode without any problems (see [issue 278](https://github.com/gitpod-io/openvscode-server/discussions/278)), does openVSCode have a logo that I can use in my blog post (see [here](https://jeroenverhoeckx.com/build-your-own-online-ide.html))? Or can I just use the logo of Gitpot (as I do now)?

**Comment by akosyakov** [comment]
We don't have logo, using Gitpod logo in this context will be confusing.

**Reply by Verhoeckx** [reply]
Do you, or somebody from Gitpod, have an idea for a logo?
The logo of Gitpod but in a different color, and with the text OpenVSCode?

**Comment by Verhoeckx** [comment]
Okay, thanks, I will remove to logo and replace it with something else!
